src/styles/blocks/variables.scss

```
@@ -0,0 +1,18 @@
+$text-size: 30px;
+$day-size: 100px;
+$gap-size: 1px;
+$calendar-width: $day-size * 7 + $gap-size * 6;
```

```suggestion
$gap-size: 1px;
$week-length: 7;
$gap-counts: $week-length - 1;
$calendar-width: $day-size * $week-length + $gap-size * $gap-counts;
```
Dont use magic numbers)